src/components/BaseLayerToggle.tsx

```diff
@@ -62,32 +78,134 @@ const InnerBackground = styled.span<InnerImageProps>`
   `};
 
   @media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
-    background: ${({
-      type,
-    }) => `url(${BASE_LAYER_STYLE[type].background[1]}) no-repeat top left /
+    background: ${({ type }) => `url(${
+      BASE_LAYER_STYLE[type as BaseLayerType].background[1]
+    }) no-repeat top left /
       120px 44px`};
   }
 `
+const Menu = styled(ContextMenu)`
+  & > button {
+    height: 44px;
```

Comment:
Could we resolve this with `themeSpacing`?

Reply:
I think we shouldn't use themeSpacing for dimensions, only paddings and margins

Reply:
Fair point.